pytripgui/view/field.ui

```diff
@@ -35,10 +35,10 @@
    <item>
     <widget class="QWidget" name="Field" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
-      <item row="1" column="2">
```

Comment:
Why ui file was so heavily modified ? Removing one object shouldn;t change it so much.

Reply:
I think it is all because of Qt Creator mess everything around. The only thing i did was to remove one element, and shrink minimum height.